tests/IntegrationTest.php

```diff
@@ -57,7 +54,7 @@ public function testPingCtorWithExplicitHttpUrlSendsRequestToGivenHttpUrlWithBas
         $client = new Client(null, str_replace('tcp://', 'http://', $socket->getAddress()) . '/base/');
         $client->ping();
 
-        $value = \Clue\React\Block\await($deferred->promise(), $loop, 1.0);
+        $value = \React\Async\await(\React\Promise\Timer\timeout($deferred->promise(),1.0));
```

Comment:
`timeout()` function does not have an explicit dependency as per https://github.com/clue/reactphp-docker/pull/77#discussion_r999436868, WDYT?

Reply:
Fixed up